.travis.yml

```diff
@@ -1 +1,24 @@
 language: go
+
+env:
+  - WITH_COVERAGE=true
+  - GOFLAGS='-race'
+
+go_import_path: github.com/google/monologue
+
+matrix:
+  fast_finish: true
+
+install:
+  - go get -d -t ./...
+  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.13.2
+
+script:
+  - set -e
+  - ./scripts/presubmit.sh ${WITH_COVERAGE:+--coverage}
+  - set +e
+
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
Similarly a comment for this would be useful too - the reason I mention this is because when Pierre looked at this he was puzzled by this line and the curl one above seemingly trying to do similar things but using different bash syntax to do it.  When I dug around a bit it became apparent that these lines have been copy pasted from instructions elsewhere.

Reply:
Done.